### DISCUSSION TITLE
DeprecationWarning: Use of deprecated folder mapping "./public/" in the "exports" field module

### DISCUSSION THREAD

**Discussion by tigerblue77** [requarks/wiki#5247] [Error / Bug Report] [answered]
Hello,

had the following error while upgrading from v2.5.278 to 2.5.279 :

![image](https://user-images.githubusercontent.com/37409593/166212223-20945c33-af6c-4f2d-91c2-2431fda459b1.png)

> (node:970427) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./public/" in the "exports" field module resolution of the package at /var/www/wiki.tigerblue77.fr.nf/node_modules/extract-files/package.json.
Update this package.json to use a subpath pattern like "./public/*".
(Use `node --trace-deprecation ...` to show where the warning was created)

Using :
- Debian 11.3
- NodeJS v16.15.0

**Comment by NGPixel** [comment, marked as answer]
This warning can safely be ignored.

**Reply by tigerblue77** [reply]
Of course, but this should be taken into consideration for the next versions, that's why I gave this feedback, thanks :)

**Reply by NGPixel** [reply]
It's already taken cared of in v3
